lib/ansible/modules/windows/win_data_deduplication.ps1

```diff
@@ -0,0 +1,174 @@
+#!powershell
+
+# Copyright: 2019, rnsc(@rnsc) <github@rnsc.be>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+
+$ErrorActionPreference = "Stop"
+
+$params = Parse-Args $args -supports_check_mode $false
```

Comment:
If we were going to accept this module I would highly recommend you support check mode, it should be fairly trivial to implement.